Gemfile

```diff
@@ -27,17 +27,21 @@ gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 3.0'
+gem 'minitest', '~> 5.10.3'
 
 gem 'devise'
 gem 'bootstrap', '~> 4.0.0'
 
+
+
```

Comment:
nitpick: 👾

Reply:
Hmm, I see a few possible issues here. I'm not sure which thing you're point out. Should I address all bullet points below?

- I didn't include a version # for devise... I can
- there are lines above and below this code block. Should I stack all the gems without any line spacing between - except to distinguish the groups at the end?

Also, there are a few gems present that I'm not sure if I need like coffee-rails and jbuilder. 
- I've only used jbuilder when building an API. Does it have other functions?
- As for minitest - I'm planning to phase that out soon, but the existing tests need it

Reply:
I was pointing out that there was already a blank line and 2 more were added in this commit. I'm not sure 3 blank lines are necessary for separating logical / grouped bits of code.

> I didn't include a version # for devise

Not necessary because the version can be found in the `Gemfile.lock` file, though it wouldn't hurt to specify it here so a `bundle update` wouldn't accidentally upgrade it and break all the things.

> there are lines above and below this code block. Should I stack all the gems without any line spacing between - except to distinguish the groups at the end?

I was pointing the spacing out yes. I don't think you need to remove all of them, but I also think 3 is a tad excessive. But totally a stylistic choice, so that's why I prefaced with `nitpick`.

> I've only used jbuilder when building an API. Does it have other functions?

That's essentially what it's good for -- for defining JSON responses. Since we're not building that (yet), I'd say you're fine removing it.

Hope that helps to clarify the meaning of 👾 emoji!